packages/ui/src/components/Radio/radiogroup.tsx

```diff
@@ -15,16 +16,22 @@ type RadioGroupOptionType = Pick<RadioProps, 'value' | 'label'> & {
 
 type RadioGroupType = {
     name: string
-    icon?: React.ReactElement
     options: Array<RadioGroupOptionType>
 }
 
+// TODO(DOMA-5430): breaking changes (v2.0.0)
 export type RadioGroupProps = {
-    icon: React.ReactElement
-    groups: Array<RadioGroupType>
-} & Pick<DefaultRadioGroupProps, 'value' | 'onChange' | 'disabled'>
+    /**
+     * @deprecated should use "children" and "RadioGroup.ItemGroup" instead of "groups". "groups" will be removed in the next major release (v2).
+     */
+    groups?: Array<RadioGroupType>
+} & Pick<DefaultRadioGroupProps, 'value' | 'onChange' | 'disabled' | 'children'>
 
-const GroupWithIcon: React.FC<RadioGroupType> = ({ name, options, icon }) => {
+type CompoundedComponent = React.FC<RadioGroupProps> & {
+    ItemGroup: typeof GroupWithIcon
+}
+
+const GroupWithIcon: React.FC<RadioGroupType> = ({ name, options }) => {
```

Comment:
Maybe we should rename with to `ItemGroup` as well?

Reply:
fixed it